iids/analyzer/views.py

```diff
@@ -3,12 +3,37 @@
 
 # Create your views here.
 #loading our trained model
-print("Keras model loading.......")
+print(" Model loading.......")
 model = load_model('attack_labe.hdf5') #after training #TODO
 print("Model loaded!!")
 
-    def request():
-        return HttpResponse("<h1>Intelligent Intrusion Detection System</h1>")
-         return HttpResponse("The predicted attacked label is {}".format(label)) #label will be imported after finishing dvevelopment of analyzer
-         
-         
\ No newline at end of file
+
+#class Ml_Algo and functions 
```

Comment:
I think we can put together all algorithms in a single class. any thoughts?

Reply:
No. It's not a good idea. we should have a separate class per algorithm

Reply:
Ok then 👍